src/ray/gcs/gcs_server/gcs_table_storage.h

```diff
@@ -274,6 +275,14 @@ class GcsWorkerFailureTable : public GcsTable<WorkerID, WorkerFailureData> {
   }
 };
 
+class GcsInternalConfigTable : public GcsTable<UniqueID, StoredConfig> {
```

Comment:
How about create a kV table? Both key and value type are string. There are other scenarios that need to use it, such as saving GCS server address. The GcsKVTable can use StoreClient directly.

Reply:
This is probably a good idea but not sure it should be a blocker for this PR